packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/inline-links.twig

```diff
@@ -0,0 +1,15 @@
+{% if inlineLinks.links %}
+  <section class="ma__inline-links {{ (inlineLinks.background)?"ma__inline-links--"~inlineLinks.background:"" }}">
```

Comment:
```suggestion
  <div class="ma__inline-links {{ (inlineLinks.background)?"ma__inline-links--"~inlineLinks.background:"" }}">
```
suggest to use div instead of section - section tag requires a heading https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
done